homeassistant/components/zwave/helper.py

```diff
@@ -0,0 +1,11 @@
+"""Helpers for zwave specific config validation using voluptuous."""
```

Comment:
Don't name this file `helper`, that is way too generic for something that only contains voluptuous helpers. If you want to keep calling it `helper`, rename the function to `validate_zwave_network_key`